kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/TopicEnforcer.java

```diff
@@ -89,11 +95,12 @@ public TopicEnforcer(
                 t -> {
                   Result result = configDrift.check(t, existing.get(t.getName()), type);
                   if (logResults && !Result.NO_DRIFT.equals(result)) {
-                    LOG.info(
-                        "Found {} for topic {}, in {} drift detection mode",
-                        result,
-                        t.getName(),
-                        type);
+                    String msgFmt = "Found {} for topic {}, in {} drift detection mode";
```

Comment:
have you tried a switch statement here? Might be cleaner since we are dealing with at least 3 different result types now

Reply:
done